### PR TITLE
add log collector creds secret to role

### PR DESCRIPTION
**Pull request by sigmarkarl**
The bigdata-spark-watcher needs to be able to access the creds for the log collector s3 bucket

# Jira Ticket

[BGD-4392](https://spotinst.atlassian.net/browse/BGD-4392)

# Checklist:
- [x] I have filled relevant self assessment ([NodeJS](https://docs.google.com/forms/d/e/1FAIpQLSfl14u9AOBAmxVJ272tvO7XNuXE-EMvWaGcaRZalu1UAKB7RA/viewform), [Frontend](https://docs.google.com/forms/d/e/1FAIpQLSdiBPNKH81w_EkavihVL8Uwb0j7tP8PwJmLFYm2nCOQxz-1qw/viewform), [Backend](https://docs.google.com/forms/d/e/1FAIpQLSed_PsTJ5-XIWkFL6BSDE2AQRBVPmwc3PAmHZkUn-erzVI37Q/viewform?usp=sf_link))
- [x] I have run ESlint on my changes and fixed all warnings and errors (**NodeJS & Frontend Services**)
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have validated all the requirements in the Jira task were answered
- [x] I have all neccessary approvals for the design/mini design of this task
- [ ] I have approved the API changes and granular permission patterns (documentation subtask) (**For public services only**) 

[BGD-4392]: https://spotinst.atlassian.net/browse/BGD-4392?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: charts/bigdata-spark-watcher/values.yaml [hunk 1]
```diff
@@ -27,6 +27,10 @@ envVars:
     value: 6m
   - name: APP_SYNC_REVERSE_GHOST_GRACE_PERIOD
     value: 30s
+  - name: KUBE_EVENT_LOG_COLLECTION_BUCKET
+    value: "spot-bigdata-logcollector-"
+  - name: KUBE_EVENT_LOG_COLLECTION_ENABLED
```
Comment (alextarasov-spot) [inline charts/bigdata-spark-watcher/values.yaml:32]: it shouldn't be in the `envVars`, it should be just a value, like `logLevel` for example. And then should map it in the deployment template, e.g.

```yaml
      containers:
        - name: manager
          args:
          - --enable-event-log-collector
          - "{{ .Values.eventLogCollectorEnabled}}"
```

in that case, we will be able to set that param in the bigdataenvironments definition

File: charts/bigdata-spark-watcher/values.yaml [hunk 2]
```diff
@@ -73,6 +81,9 @@ resources:
     cpu: 2000m
     memory: 2000Mi
 
+eventLogCollectorEnabled:
+  true
```
Comment (alextarasov-spot) [inline charts/bigdata-spark-watcher/values.yaml:85]: let's set it to false by default. we will enable it using custom environments per customer.

File: charts/bigdata-spark-watcher/values.yaml
Comment (alextarasov-spot) [inline charts/bigdata-spark-watcher/values.yaml:1]: update tag version `0.4.5-78b84f0c` -> NEW_VERSION

**Comment by alextarasov-spot** [comment]
bump chart version https://github.com/spotinst/bigdata-charts/blob/ab3ba862e0d94a0afc6673fe108873c5195846a6/charts/bigdata-spark-watcher/Chart.yaml#L6

**Review by raggivald — APPROVED** [review]

**Pull request merged**